index.html

```diff
@@ -45,6 +45,133 @@ <h2 class="tx-center">Full-Stack Software Developer</h2>
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8" type="button">See Project</button>
+      <div class="blur-elipse"></div>
+    </div>
+    <div class="g-container">
+      <img class="g-item1" src="img/Snapshoot-Portfolio.png" alt="Imagen de prueba">
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8" type="button">See Project</button>
+      <div class="blur-elipse-2"></div>
+    </div>
+    <div class="g-container">
+      <img class="g-item1" src="img/Snapshoot-Portfolio.png" alt="Imagen de prueba">
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8" type="button">See Project</button>
+      <div class="blur-elipse-3"></div>
+    </div>
+    <div class="g-container">
+      <img class="g-item1" src="img/Snapshoot-Portfolio.png" alt="Imagen de prueba">
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8 mr-b" type="button">See Project</button>
+      <div class="blur-elipse-4"></div>
+    </div>
+  </section>
+  <!--My Work section-->
+  <hr>
+  <!--About Me section-->
+  <section id="aboutMe" class="grid-container-2">
+    <img class="g-item9" src="img/illustration-about-me.png" alt="Imagen de prueba">
```

Comment:
- [ ] In order to stick to the design as much as possible, kindly remove this image, as it is for another template and it is not specified in this template.

Reply:
- [ ] Kindly implement all changes requested by previous reviewer, this image is still in your project 👍🏿